Assets/Scripts/Player/PlayerModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
```

Comment:
I think you could just set the animation states from the PlayerMovement class instead of having a separate class to do that.